pom.xml

```diff
@@ -85,6 +97,11 @@
             <artifactId>utils</artifactId>
             <version>${pay-java-commons.version}</version>
         </dependency>
+        <dependency>
+            <groupId>uk.gov.service.payments</groupId>
+            <artifactId>model</artifactId>
```

Comment:
looks like this is no longer used by the PR but I assume it will be used later down the line to have standard responses from backends